plugins/modules/proxmox_disk.py

```diff
@@ -260,10 +260,10 @@
     description:
       - The ISO image to be mounted on the specified in O(disk) CD-ROM.
       - O(media=cdrom) needs to be specified for this option to work.
-      - Image string format
-      - C(<STORAGE>:iso/<ISO_NAME>) to mount ISO.
-      - C(cdrom) to use physical CD/DVD drive.
-      - C(none) to unmount image from existent CD-ROM or create empty CD-ROM drive.
+      - Image string format:
```

Comment:
Sorry, this should better be
```suggestion
      - "Image string format:"
```
so that this isn't a list element which contains a dictionary with key "Image string format" and value "None" :)